src/components/dictionaryConcepts/components/MapType.jsx

```diff
@@ -3,12 +3,12 @@ import PropTypes from 'prop-types';
 
 const mapType = (props) => {
   const {
-    index, map_type, map_types, updateEventListener, url,
+    index, map_type, map_types, updateEventListener, url, source,
   } = props;
   return (
     <select
       tabIndex={index}
-      defaultValue={map_type}
+      value={source !== 'CIEL' ? 'NARROWER-THAN' : map_type}
```

Comment:
Shouldn't this be already in a constant?

Reply:
I have fixed it and added the constant `INTERNAL_MAPPING_DEFAULT_SOURCE` used for `'CIEL'`